ff_dev_support.c

```diff
@@ -20,8 +19,8 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * https://www.FreeRTOS.org
- *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
```

Comment:
The URLs should be:

https://FreeRTOS.org
https://github.com/FreeRTOS

which are similar to the FreeRTOS-Kernel urls:
https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/list.c#L22

Applies to all locations

Reply:
And not the complete URL on github?

https://github.com/FreeRTOS/Lab-Project-FreeRTOS-FAT ?

Reply:
I will have to assume no, since in the kernel files it is only https://github.com/FreeRTOS and not .....FreeRTOS/FreeRTOS-Kernel

Reply:
That sounds logical. I updated all copyright notices.